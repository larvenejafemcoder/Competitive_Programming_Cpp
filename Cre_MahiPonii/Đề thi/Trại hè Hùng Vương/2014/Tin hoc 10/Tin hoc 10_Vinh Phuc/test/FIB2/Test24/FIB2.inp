61
33 3804246
8 18
12 197
7 4
25 12961
41 110316126
5 3
30 1072156
30 785946
14 13
12 131
32 2574519
38 19132017
6 9
41 165335308
18 4042
23 877
34 3125378
44 911500637
25 7745
12 40
10 21
8 14
26 81970
40 49599134
8 28
6 9
38 22115392
22 23260
41 161057573
35 11909347
18 602
38 21747290
5 5
36 2706768
40 22066098
17 629
29 87553
21 2983
35 4197836
10 73
19 5858
18 1224
36 14199068
3 3
12 129
20 9917
19 3263
0 1
29 17555
13 37
22 11112
40 85913707
42 182783711
15 540
29 750176
42 334513020
43 245052417
4 2
21 2953
4 4
