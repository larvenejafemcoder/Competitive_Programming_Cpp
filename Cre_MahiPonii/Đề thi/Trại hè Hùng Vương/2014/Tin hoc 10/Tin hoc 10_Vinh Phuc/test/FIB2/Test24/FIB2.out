1453093
7
75
2
4951
42137011
1
409527
300205
5
50
983379
7307780
4
63152468
1544
335
1193788
348162263
2958
15
8
5
31310
18945183
11
4
8447328
8885
61518519
4548966
230
8306726
2
1033893
8428500
240
33442
1140
1603431
28
2238
468
5423561
1
49
3788
1246
1
6706
14
4245
32816116
69817165
206
286542
127772604
93601694
1
1128
2
